components/sidebar.jsx

```diff
@@ -100,6 +104,12 @@ export default class Sidebar extends React.Component {
     }
 
     componentDidMount() {
+        fetchMyChannelsAndMembers(this.state.currentTeam.id)(dispatch, getState).then(
+            () => {
+                loadStatusesForChannelAndSidebar();
+                loadProfilesForSidebar();
+            }
+        );
```

Comment:
Can we create an action that wraps all these into one and then make it use `await` instead of `.then()`? Should make it easier to use

Reply:
Sure, I will make the change later today.